pkg/doc/did/doc.go

```diff
@@ -597,26 +670,40 @@ func decodePublicKeyJwk(jwkMap map[string]interface{}, publicKey *PublicKey) err
 	return nil
 }
 
-func (r *rawDoc) ParseContext() []string {
+func (r *rawDoc) ParseContext() ([]string, string) {
 	switch ctx := r.Context.(type) {
 	case []interface{}:
 		var context []string
+
+		var base string
+
 		for _, v := range ctx {
-			context = append(context, v.(string))
+			valueOfString, ok := v.(string)
```

Comment:
Extract the code into a separate method? Use type switch?

Reply:
done